docs/pt/README-PT.md

```diff
@@ -0,0 +1,107 @@
+<!-- Deep Learning (with PyTorch)
+-->
+# Aprendizagem Profunda (com PyTorch) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Atcold/pytorch-Deep-Learning/master)
+
+<!-- This notebook repository now has a [companion website](https://atcold.github.io/pytorch-Deep-Learning/), where all the course material can be found in video and textual format.
+-->
+Este repositório de *notebooks* possui um [página web complementar](https://atcold.github.io/pytorch-Deep-Learning/es/), onde todo o material do curso pode ser encontrado em formato textual e em vídeo.
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic - Brazilian Portuguese -->
+[🇬🇧](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/README.md) &nbsp; [🇨🇳](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/zh/README-ZH.md) &nbsp; [🇰🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ko/README-KO.md) &nbsp; [🇪🇸](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md) &nbsp; [🇮🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md) &nbsp; [🇹🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) &nbsp; [🇯🇵](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) &nbsp; [🇸🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) &nbsp; [🇧🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/pt_BR/README-PT_BR.md)
```

Comment:
Wrong file name in the link.

Reply:
Just to see if I understood it right, would be updating:

- The wrong file name (from `README-PT_BR.md` to `README-PT.md`.
- Including links to `PT` into all README files, regardless of language.

Correct?

Reply:
Yes, that's correct. This file is called `README-PT.md`, correct? You're linking to a non existent `README-PT_BR.md`. And yes, all the `README.md`s need to have a pointer to the Portuguese version.